.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+node_modules/prettier/bin-prettier.js --write "src/**/*.scss" "dist/dev/mbuttons.css"
```

Comment:
Needs a decision on adding `git add .` to commit all sorts of staged files.

Otherwise, keeping it this way to implement a fine-grained control on commit messages and id.